.jenkins/pipelines/standalone/azure-sdk-tests.Jenkinsfile

```diff
@@ -99,8 +102,49 @@ pipeline {
                    """
             }
         }
+        stage("Install package") {
+            when {
+                expression { params.PACKAGE_NAME != "" }
+            }
+            steps {
+
+                azureDownload(
+                    downloadType: 'container',
+                    containerName: 'mystikosreleases',
+                    includeFilesPattern: "${PACKAGE_INSTALL}",
+                    storageCredentialId: 'mystikosreleaseblobcontainer'
+                )
+
+                script {
+                    if ( params.PACKAGE_EXTENSION == "deb" ) {
+                        sh "sudo dpkg -i ${PACKAGE_INSTALL}"
+                    } else {
+                        sh """
+                           rm -rf mystikos
+                           tar xzf ${PACKAGE_INSTALL}
+                           """
+                    }
+                }
+            }
+        }
         stage('Run Azure SDK tests') {
+            environment {
+                MYST_INSTALLED_BIN = "${ params.PACKAGE_EXTENSION == 'deb' ? '/opt/mystikos/bin' : '\${WORKSPACE}/mystikos/bin' }"
```

Comment:
May be cleaner to instead use PATH when installing from deb / tar so you don't need to do this linking? Unless there are issues other than calling the binaries

Example `PATH = "/opt/mystikos/bin:\$PATH"`

Reply:
Also, it would be better to have put this in it's own stage so you don't need to echo titles out and be slightly easier to navigate in Jenkins Blue Ocean

Reply:
The debian/tar packages are built with three binaries: `myst`, `myst-gdb`, and `myst-appbuilder`. But tests actually use four binaries, the three mentioned above and `myst-lldb`, which isn't included in the released packages.

Tests allow paths to the `myst`, `myst-gdb`, and `myst-lldb` binaries, but not to `myst-appbuilder`.

So because `myst-lldb` is being referenced in the `build/bin` folder, and `myst-appbuilder` is being referenced in the `scripts` folder, I figured the most consistent way to reference different binaries is to let the tests reference the original source locations, but just have symbolic links to the actual binaries that are to be used.

Reply:
Also, yes, moved to the stage above.